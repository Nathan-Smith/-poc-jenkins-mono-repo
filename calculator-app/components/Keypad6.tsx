import { KeypadButton } from './KeypadButton'

export function Keypad6(): JSX.Element {
  return (
    <KeypadButton testid="keypad-6" gridArea="keypad-6">
      6
    </KeypadButton>
  )
}
