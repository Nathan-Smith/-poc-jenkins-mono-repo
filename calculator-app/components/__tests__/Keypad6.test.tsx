import React from 'react'
import renderer, { act, ReactTestRenderer } from 'react-test-renderer'

import { Keypad6 } from '../Keypad6'

describe('Keypad6', () => {
  test('renders', async () => {
    let component: ReactTestRenderer | undefined
    act(() => {
      component = renderer.create(<Keypad6/>)
    })

    expect(component?.toJSON()).toMatchSnapshot()
  })
})
