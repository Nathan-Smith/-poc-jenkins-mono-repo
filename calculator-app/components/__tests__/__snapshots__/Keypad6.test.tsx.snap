// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Keypad6 renders 1`] = `
<button
  className="mdc-button mdc-button--outlined keypadButton"
  data-testid="keypad-6"
  onClick={[Function]}
  style={
    Object {
      "gridArea": "keypad-6",
    }
  }
>
  <span
    className="mdc-button__ripple"
  />
  <span
    className="mdc-button__label"
  >
    6
  </span>
</button>
`;
